<launch>
  <node name="sensor_main" pkg="sensor_node" type="sensor_main" output="screen">
      <param name="ANGLE_COEF"    type="double" value="1.95"/>
      <param name="CUT_OFF_RATIO"    type="double" value="2"/>
      <param name="ROBOT_SPEED"   type="double" value="0.7"/>
      <param name="MIN_ANGLE"    type="int" value="10"/>
      <param name="MAX_DIS"   type="double" value="1.4"/>
      <param name="SIDE_DIS"   type="double" value="0.45"/>
      <param name="MAX_ANGLE"    type="int" value="75"/>
      <param name="MIN_DIS"   type="double" value="0.05"/>
      <param name="DECEL"   type="double" value="0.0"/>
  </node>
  
</launch>
